docs/content/docs/reference/nextjs/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Introduction'
+---
+
+It is possible to use [`@nhost/react`](/reference/react) in any Next.js page that would be configured to render on the client side.
+
+When rendering a page from the server side, Next.js needs to get some information from the client to determine their authentication status. Such a communication is only available from cookies, and the the Nhost client has been designed to enable such a mechanism.
+
+The cookie is `same-site` and is used only to store the refresh cookie and the expiration date of the JWT so the browser can proactively ask for a new JWT before it expires.
+
+## Signing in
+
+There is no need to activate SSR in the sign-in page as there is not yet any session information to capture. As a result, it is recommended to code it as a classic static page.
+
+The intitial authentication would then work like this diagram:
```

Comment:
"There is no need to activate SSR on the sign-in page as there is no session information to capture. As a result, it is recommended to code it as a classic static page.

The initial authentication would then work like this diagram:"